titlecase/tests.py

```diff
@@ -303,6 +303,30 @@
         "ýæ ñø",
         "Ýæ Ñø",
     ),
+    (
+        "Mr",
+        "Mr",
+    ),
+    (
+        "mr",
+        "Mr",
+    ),
+    (
+        "Mrs",
+        "Mrs",
+    ),
+    (
+        "Ms",
+        "Ms",
+    ),
+    (
+        "Mss",
+        "MSS",
+    ),
+    (
+        "Mr Person",
+        "Mr Person",
+    ),
```

Comment:
```suggestion
    # https://github.com/ppannuto/python-titlecase/pull/67
    (
        "Mr mr Mrs Ms Mss , Mr. and Mrs. Person",
        "Mr Mr Mrs Ms MSS , Mr. and Mrs. Person",
    ),
```

Reply:
Yes, that's much better. I don't know why I didn't think of that :D